### DISCUSSION TITLE
buddhism/core/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#27] [General]
# buddhism/core/

https://shanenull.com/buddhism/core/

**Comment by shane0** [comment]
https://youtu.be/alZaa06ZFJw?si=mVlY7leMnjT582MX

**Comment by shane0** [comment]
The three poisons in Buddhism, also known as the three kleshas, are the root causes of suffering. They are:

* **Raga (attachment)**: This is the attachment to things, people, and experiences that we believe we need to be happy. It is the desire to cling to what we have and to avoid what we don't. Raga leads to suffering because it is based on the illusion that things are permanent and that we can control them.

* **Dvesha (aversion)**: This is the aversion to things, people, and experiences that we believe are causing us pain. It is the desire to push away what we don't want and to hold onto what we do. Dvesha leads to suffering because it is based on the belief that we are separate from the world around us and that we can control our experiences.

* **Avidya (ignorance)**: This is the ignorance of our true nature and the nature of reality. It is the belief that we are separate from everything else, that we are solid and unchanging, and that we have control over our lives. Avidya leads to suffering because it prevents us from seeing things for what they really are and from living in accordance with the truth.

The three poisons are interconnected, and they all contribute to each other. For example, raga can lead to dvesha when we become attached to something and then lose it. Dvesha can lead to avidya when we become so focused on what we don't want that we lose sight of what is actually happening. And avidya can lead to raga and dvesha when we don't understand our true nature and the nature of reality.

The path to liberation from suffering is to cultivate the opposite of the three poisons. This means developing **virtue (sila)**, **concentration (samadhi)**, and **wisdom (prajna)**.

Sila is the foundation of the path. It is the practice of ethical conduct, which includes refraining from harming others, taking what is not given, and speaking falsely. By practicing sila, we create a foundation of peace and harmony in our lives.

Samadhi is the practice of concentration. It is the ability to focus the mind on the present moment and to let go of distractions. By practicing samadhi, we develop the ability to see things more clearly and to understand the true nature of reality.

Prajna is the practice of wisdom. It is the understanding of our true nature and the nature of reality. By practicing prajna, we are able to see that the three poisons are illusions and that we are not separate from the world around us.

When we cultivate sila, samadhi, and prajna, we are able to break free from the cycle of suffering and to experience true liberation.

Here are the Sanskrit words for the three poisons:

* Raga: राग

* Dvesha: द्वेष

* Avidya: अविद्या
